src/pages/case-studies/satellite/index.mdx

```diff
@@ -9,4 +9,73 @@ tags:
+
+However, software-defined satellites still face numerous difficulties in software or application packaging, distribution, deployment, upgrades, and maintenance. 
+They struggle to meet the satellite's on-orbit business requirements for rapid iteration, instant deployment, grayscale release, and continuous evolution. 
+As a result, both domestic and international efforts are trying to develop a new generation of intelligent satellites.
+
+## Breakthrough: From Nothing to Something, Breaking On-Orbit Computing Barriers
+
+On December 7, 2021, the world's first cloud-native satellite was successfully launched. As the global first cloud-native satellite, 
+it leverages container technology, microservices, and agile methods to achieve continuous application delivery through DevOps, with the following advantages:
+
+- **Rapid Deployment:** Cloud-native satellites use agile development and DevOps, enabling users to quickly develop programs, automatically deploy applications, 
+and continuously update onboard applications. This allows applications to keep up with changing demands, and even guide demand evolution.
+
+- **Elastic Scaling:** Cloud-native satellites inherently possess cloud computing characteristics, with resources that can be scaled according to actual conditions, 
+improving resource utilization.
+
+- **Security and Reliability:** Relying on container orchestration and microservices, cloud-native satellites enable onboard applications with capabilities like automatic recovery, 
+fault tolerance, and fault isolation. This helps address complex space environments, keeping applications constantly available.
+
+- **High Autonomy:** Using containerization technology, onboard applications run within containers, independent of location and environment. The dependencies, 
+configuration files, and backend services required to build applications are integrated, achieving self-management.
+
+Traditional communication and remote sensing satellites only have storage capabilities, merely storing data to be transmitted back to the ground, without onboard processing capabilities.
+By utilizing cloud-native edge computing technology, satellites can process data in space, significantly reducing transmission pressure and meeting the diverse needs of aerospace fields. 
+
+**Significance: Giving Satellites "Apps", Enabling "Business in Space and Services in Orbit"**
+
+As shown in the diagram, by reconstructing the KubeEdge system and lightweight AI inference model Sedna according to satellite characteristics, empowering satellites with cloud-native edge computing capabilities, and achieving collaborative AI inference between satellites and ground stations, the following was accomplished:
+- In remote sensing scenarios, on-orbit target identification accuracy improved by over 50%
+- Satellite data transmission volume reduced by 90%
+- Support for in-orbit continuous AI model updates
+
+![Satellite](images/satellite.png)
```

Comment:
Do you have an editable version of this image? Maybe we can also replace the Chinese in the picture with English

Reply:
I had this english one so added : ) also Elias, a lot of Indian devs were interested in these case studies so I will create english versions for other case studies as well, please review them as well : )

Reply:
Sounds great! Just go ahead!